Test/Service.cs

```diff
@@ -28,12 +28,12 @@ public static class Util {
                 case "batches":
                     return new[] {"batches", "nextToken"};
                 case "dataset":
-                    return new[] {"name", "description", "datasetId", "version", "numberOfDocuments"};
```

Comment:
No numberOfDocuments?

Reply:
Current OAS spec  does not have numberOfDocuments